.github/workflows/package.yaml

```diff
@@ -0,0 +1,21 @@
+name: release-binaries
```

Comment:
To be able to have this workflow triggered, you will have to change `release.yaml` similar to the tigrisdb repo and use the bot user's token instead of GITHUB_TOKEN:

```
- name: release
        env:
          GITHUB_TOKEN: ${{ secrets.GH_BOT_ACCESS_TOKEN }}
        run: npx semantic-release --debug
```

Reply:
Good catch.
I am adding it now for manual testing, will fix automation later.

Reply:
Sounds good, stamping